scripts/publish.sh

```diff
@@ -53,20 +60,28 @@ ARTIFACT_FILE="$(ci-pkginfo -t pkgname)-$(ci-pkginfo -t pkgsemver).tgz"
 DEPLOY_VERSION="$([[ ${ARTIFACT_FILE} =~ vuepress-site-(.*)\.tgz ]] && echo ${BASH_REMATCH[1]})"
 ARTIFACT_PATH="@okta/vuepress-site/-/${ARTIFACT_FILE}"
 
-if ! trigger_and_wait_release_promotion_task 60; then
-  echo "Automatic promotion failed..."
-  exit ${BUILD_FAILURE}
+# Only auto-promote to npm-release on main branch
+if [[ $BRANCH == "master" ]]; then
+  if ! trigger_and_wait_release_promotion_task 60; then
```

Comment:
I learned that this promotion task is what moves an artifact from `npm-topic` into `npm-release`, so it can then be consumed by the conductor production deployment pipeline.